### ISSUE TITLE
why there no avgpooling？

### ISSUE THREAD

**Issue by fire717** [Cadene/pretrained-models.pytorch#204]
Hi every I am new for pytorch.
In other Pytorch model I saw，
there must be an avgpooling between CNN (which have 4 dimension) and Linear (only 2 dimension),
And in Keras，it can be GlobalAverangePooling.

But in this repo, like Xception, I print the model the last 4 layers like this：
 ...
BatchNorm2d-218           [-1, 2048, 7, 7]           4,096
ReLU-219                         [-1, 2048, 7, 7]               0
Linear-220                        [-1, 3]                      6,147
Xception-221                    [-1, 3]                         0

There is no avgpooling,  and the 4 dimention just become 2 dimention,
anyone can explain this? Thx!

**Comment by fire717** [comment]
Saw it in the code.
